src/pool/srv_iv.c

```diff
@@ -465,6 +494,12 @@ pool_iv_ent_refresh(struct ds_iv_entry *entry, struct ds_iv_key *key,
 	if (rc)
 		return rc;
 
+	pool = ds_pool_lookup(dst_iv->piv_pool_uuid);
+	if (pool == NULL) {
+		D_WARN("No pool "DF_UUID"\n", DP_UUID(dst_iv->piv_pool_uuid));
+		return 0;
+	}
+
 	D_ASSERTF(src_iv != NULL, "%d\n", entry->iv_class->iv_class_id);
 	/* Update pool map version or pool map */
 	pool = ds_pool_lookup(src_iv->piv_pool_uuid);
```

Comment:
The reference of the first lookup will be lost, no?

Reply:
ah, right